pages/_app.jsx

```diff
@@ -10,15 +10,14 @@ export default function App({ Component, pageProps }) {
   const [games, setGames] = useState(initialGames);
   const [menuMode, setMenuMode] = useState("closed");
   const [sortingOrder, setSortingOrder] = useState("alphabetically-A-to-Z");
-
   function handleChangeSortingOrder(order) {
     setSortingOrder(order);
   }
 
   function handleCreateGame(newGame) {
     const id = uid();
     setGames([
-      ...games,
+      ...(games ? games : []),
```

Comment:
I would argue this is more readable:

```suggestion
      ...(games || []),
```

Reply:
Absolutly! I was not aware of this solution. I thought I had to use a ternary operator.